src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -6899,6 +6900,13 @@ private BoundExpression MakeMemberAccessValue(BoundExpression expr, BindingDiagn
                         CompoundUseSiteInfo<AssemblySymbol> useSiteInfo = GetNewCompoundUseSiteInfo(diagnostics);
                         var resolution = this.ResolveMethodGroup(methodGroup, analyzedArguments: null, isMethodGroupConversion: false, useSiteInfo: ref useSiteInfo);
                         diagnostics.Add(expr.Syntax, useSiteInfo);
+
+                        if (resolution.IsExtensionMember(out Symbol extensionMember))
```

Comment:
>resolution.IsExtensionMember(out Symbol extensionMember) [](http://example.com/codeflow?start=28&length=56)

Could we resolve to an indexer here, or, perhaps, an indexer group?

Reply:
I'll add some related tests (trying to access `.Item` or `.get_Item`), but I don't think we can. That would require named properties/indexers in C#.